core/math/convex_hull.cpp

```diff
@@ -2257,22 +2258,21 @@ Error ConvexHullComputer::convex_hull(const PoolVector<Vector3> &p_points, Geome
 }
 
 Error ConvexHullComputer::convex_hull(const Vector3 *p_points, int32_t p_point_count, Geometry::MeshData &r_mesh) {
-	r_mesh = Geometry::MeshData(); // clear
+	r_mesh = Geometry::MeshData(); // Clear.
 
 	if (p_point_count == 0) {
-		return FAILED; // matches QuickHull
+		return FAILED; // Matches QuickHull.
 	}
 
 	ConvexHullComputer ch;
 	ch.compute(p_points, p_point_count, -1.0, -1.0);
 
 	r_mesh.vertices = ch.vertices;
+	int vertex_count = r_mesh.vertices.size();
```

Comment:
Could do with a comment here and at the resize at the bottom of the function to explain why this is being done - it is clear in the PR text, but not in the source.